apps/web/app/Models/WorkspaceInvitation.php

```
@@ -0,0 +1,101 @@
+    public function inviter(): BelongsTo
+    {
+        return $this->belongsTo(User::class, 'invited_by');
+    }
+
+    /**
+     * Generate a unique invitation token
+     */
+    public static function generateToken(): string
+    {
+        do {
+            $token = Str::random(64);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Str' in method 'generateToken'. (Clean Code Rules)

(StaticAccess)